src/main/java/com/easyrest/pages/RestaurantsList.java

```diff
@@ -24,7 +24,7 @@ public class RestaurantsList extends BasePage {
     private final By cuisineJPRlMenuItem = By.xpath("//span [text()='japanese cuisine']");
```

Comment:
JPRl is a little bit confusing

Reply:
I agree with Vadym, please provide a meaningful name.